css/legacy.scss

```diff
@@ -20,5 +20,18 @@
  *
  */
 
-@import '../../../core/css/variables.scss';
+
+/**
+ * Variables
+ */
+$color-main-background: #fff;
+$color-primary-text: #fff;
+$color-error: #e9322d;
+$color-warning: #fbd850;
+$color-warning-light: nc-lighten($color-warning, 15%);
+$color-success: #46ba61;
+$color-lightgrey: nc-darken($color-main-background, 4%);
+$color-grey: nc-darken($color-main-background, 7%);
+$color-darkgrey: nc-darken($color-main-background, 32%);
+
```

Comment:
Why do you add the variables.scss content here?
The scss compilator automatically add this to any file you load with add_style :)

Reply:
Not for stable11, but we can drop this since we will only support version 12 or later in the next release.

Reply:
Yes, I asked because the info.xml says min version to 12. :)